src/controllers/check.company.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { NextFunction, Request, Response } from "express";
+
+export const route = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+
```

Comment:
I know it's obvious, but when I add code like this I tend to qualify it with a TODO comment, something like:

   // TODO Stub implementation to return dummy company details until they have been correctly retrieved

Up to you.